index.js

```diff
@@ -142,3 +142,31 @@ contactform.addEventListener("submit", (e) => {
     e.preventDefault();
   }
 });
+
+const input = document.getElementById("username");
+const email = document.getElementById("email");
+const inputs = document.querySelectorAll(".nameinput, .emailinput");
+const obj = {};
+
+for (let i = 0; i < inputs.length; i += 1) {
+  inputs[i].addEventListener("input", () => {
+    obj.name = input.value;
+    obj.email = email.value;
+    localStorage.setItem("formdata", JSON.stringify(obj));
+  });
+}
+
+contactform.addEventListener("submit", () => {
+  obj.name = input.value;
+  obj.email = email.value;
+  localStorage.setItem("formdata", JSON.stringify(obj));
+});
```

Comment:
- [ ] As requested by the previous reviewer in adding the functionality to save the data to the local storage, you are required to save all the data from the form fields which includes `name, email & message`, currently your project only saves the `name & email`, but from the image below saving all data including `message` so I kindly suggest you add this to the data being saved to local storage. 👍 
- > * ## Project requirement
![dt](https://github.com/enocol/MobileFirst/assets/89055502/e0bb4a70-f73e-468e-b8c0-7d4931dbd6f1)